common/test/acceptance/pages/lms/teams.py

```diff
@@ -102,3 +103,42 @@ def header_topic_description(self):
     def team_cards(self):
         """Get all the team cards on the page."""
         return self.q(css='.team-card')
+
+
+class TeamPage(CoursePage, PaginatedUIMixin):
+    """
+    The page for a specific Team within the Teams tab
+    """
+    def __init__(self, browser, course_id, team):
+        """
+        Set up `self.url_path` on instantiation, since it dynamically
+        reflects the current topic.  Note that `topic` is a dict
+        representation of a topic following the same convention as a
+        course module's topic.
+        """
+        super(TeamPage, self).__init__(browser, course_id)
+        self.team = team
+        self.url_path = "teams/#teams/{topic_id}/{team_id}".format(
+            topic_id=self.team['topic_id'], team_id=self.team['id']
+        )
+
+    def is_browser_on_page(self):
+        """Check if we're on the teams list page for a particular team."""
+        has_correct_url = self.url.endswith(self.url_path)
+        # The "teams-main" class is not unique to this view, but currently there is nothing else on the
+        # page except for the discussion module.
+        team_view_present = self.q(css='.teams-main').present
```

Comment:
I've introduced a ``team-profile`` class on the outermost div, and changed this test to look for it.